.editorconfig

```diff
@@ -0,0 +1,24 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# PHP files
+[*.php]
+indent_style = space
+indent_size = 4
+
+# MD files
+[*.md]
+indent_style = space
+indent_size = 4
```

Comment:
Do we not have 2 spaces for indenting in `.md` files - e.g. in bulleted lists - or is the indentation for lists something different?

Reply:
According to these style guides, it's 4 spaces (for code and lists where we do not use use hanging indent determined by the number of digits):

- https://google.github.io/styleguide/docguide/style.html#nested-list-spacing
- https://guides.github.com/features/mastering-markdown/
- https://cirosantilli.com/markdown-style-guide/#indentation-of-content-inside-lists

This setting in the `.editorconfig` only determine what the tab key does, not what a linter might expect.

Reply:
OK.